src/gtk/textctrl.cpp

```diff
@@ -124,6 +124,57 @@ static void wxGtkTextApplyTagsFromAttr(GtkWidget *text,
         }
     }
 
+    if ( attr.HasFontUnderlined() )
+    {
+        PangoUnderline pangoUnderlineStyle;
```

Comment:
Don't we get (spurious) warnings about this being uninitialized?

Reply:
No, all cases where covered in the switch. But for consistency I initialized the values and added `default` to the switches to prevent `-Wswitch` warnings.